### PR TITLE
OSS-PAID bundle docs

### PR DESCRIPTION
**Pull request by dulanism**
JIRA - https://issues.sonatype.org/browse/NEXUS-10117
CI - TBD

### CODE REVIEW

File: chapter-configuration.asciidoc [hunk 1]
```diff
@@ -901,8 +901,9 @@ The paid license is a special +.lic+ file that you upload to the 'Licensing' fea
 * Click 'I agree' to the terms stated in the End User License Agreement.
 * Click 'Authenticate' to complete the upload.
 
-After the file is selected and uploaded, the 'License type' field will display the corresponding bundle 
-you purchased.
+After the file is successfully uploaded, you will receive a notification to restart the repository manager. After
+restart the 'License type', shown in the 'Licensing' panel, will display the paid bundle corresponding to the
```
Comment (joedragons) [inline chapter-configuration.asciidoc]: As hinted in your ticket comment I think "the 'License type'" should either be "the 'License type' field" or just "'License type'" (no the).  I think the word field is fine.  If I misunderstood your Q, let me know.

Comment (kellyrob99) [inline chapter-configuration.asciidoc]: I think perhaps a better description might be "will display the features associated with your license", but we might want to defer to sales nomenclature - @dulanism Care to ask how they refer to licensed features?

Comment (dulanism) [inline chapter-configuration.asciidoc]: Will do @kellyrob99

**Comment by joedragons** [comment]
@dulanism can you include the update to https://github.com/sonatype/nexus-internal/commit/19842e4dabe4e8fd361b3ce76c5527c8e56595eb in this PR?  Otherwise I can make another edit once this is clear.  Let me know if adjustment is unclear, thanks.

**Comment by joedragons** [comment]
So after building it out and restoring OSS, I found what I had guessed to be true.  "Available in Nexus Repository Manager only" no longer applies.  As written, this section applies to both OSS and PRO.  The only way I think it'd be different is if we discussed the license details which I believe we talked about specifically not doing when we first created this section (before this update).

**Comment by joedragons** [comment]
+1 after comments are addressed.  Happy to rereview if you like, as usual.

**Comment by kellyrob99** [comment]
+1

**Pull request merged**